pkg/reconciler/knativeeventing/knativeeventing.go

```diff
@@ -148,7 +147,7 @@ func (r *Reconciler) ensureFinalizerRemoval(_ context.Context, _ *mf.Manifest, i
 func (r *Reconciler) install(ctx context.Context, manifest *mf.Manifest, ke *eventingv1alpha1.KnativeEventing) error {
 	logger := logging.FromContext(ctx)
 	logger.Debug("Installing manifest")
-	return common.Install(manifest, version.EventingVersion, &ke.Status)
+	return common.Install(manifest, common.GetLatestRelease(kcomponent), &ke.Status)
```

Comment:
Why fetch this twice? I'd probably store the determined version in the reconciler's state until we actually move the manifest loading here. Same for the serving component.